index.html

```diff
@@ -1 +1,36 @@
-<!-- JavaScript課題①-3 -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>task1-3</title>
+</head>
+<body>
+  <script>
+    const taskList = ['掃除','買い物','散歩'];
```

Comment:
`,`の後ろにはスペースを空けましょう。
```suggestion
    const taskList = ['掃除', '買い物', '散歩'];
```